pkgs/by-name/li/ligolo-ng/package.nix

```
@@ -2,38 +2,58 @@
   lib,
   buildGoModule,
   fetchFromGitHub,
+  pkgsCross ? { },
 }:
 
-buildGoModule rec {
+buildGoModule (finalAttrs: {
   pname = "ligolo-ng";
   version = "0.8.2";
 
   src = fetchFromGitHub {
-    owner = "tnpitsecurity";
+    owner = "nicocha30";
     repo = "ligolo-ng";
-    tag = "v${version}";
+    tag = "v${finalAttrs.version}";
     hash = "sha256-ND0SFB0xj4WK6okNzChZWfK5bhNc4PTWuZoq/PodTW0=";
   };
 
   vendorHash = "sha256-oc85xNPMFeaPC7TMcSh3i3Msd8sCJ5QGFmi2fKjcyvk=";
 
-  postConfigure = ''
-    export CGO_ENABLED=0
-  '';
+  env.CGO_ENABLED = 0;
+
+  subPackages = [
+    "cmd/agent"
+    "cmd/proxy"
+  ];
 
   ldflags = [
     "-s"
     "-w"
     "-extldflags '-static'"
   ];
```

Nit:
```suggestion
  ldflags = [
    "-s"
    "-w"
    "-extldflags '-static'"
    "-X main.version=${version}"
  ];
```

Then `ligolo-agent --version` will output the correct version string